src/test/java/seedu/address/logic/commands/ClearCommandTest.java

```diff
@@ -17,7 +17,7 @@ public void execute_emptyAddressBook_success() {
         Model model = new ModelManager();
         Model expectedModel = new ModelManager();
 
-        assertCommandSuccess(new ClearCommand(), model, ClearCommand.MESSAGE_SUCCESS, expectedModel);
+        assertCommandSuccess(new ClearCommand(), model, ClearCommand.MESSAGE_FAILURE, expectedModel);
```

Comment:
Referring to my previous comment, this could then be an `assertThrows` test since it will throw `CommandException` should you decide to use an exception instead of the general `CommandResult`